array-manipulation-1/src/compact.js

```diff
@@ -1,14 +1,17 @@
 /* exported compact */
 
+// set an array for the truthy values
+// go though every item
+// check if its falsy
+// if it isNaN, continue
+// else
+// push the value into the array
+// then return the array
 function compact(array) {
   var truthyArray = [];
   for (var i = 0; i < array.length; i++) {
     if (Boolean(array) === false || Boolean(array[i]) === false) {
```

Comment:
Okay, I see where you are going! So, in this case, you don't really want to find the falsy values since you don't do anything with them except `continue` forward, how can you check if a value is truthy?